<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DraftPolarArrayTaskPanel</class>
 <widget class="QWidget" name="DraftPolarArrayTaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Polar array</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QGroupBox" name="main_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="grid_values">
        <item row="0" column="0">
         <widget class="QLabel" name="label_angle">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Gui::QuantitySpinBox" name="spinbox_angle">
          <property name="toolTip">
           <string>Sweeping angle of the array</string>
          </property>
          <property name="unit" stdset="0">
           <string notr="true"/>
          </property>
          <property name="value">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_number">
          <property name="text">
           <string>Number of elements</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinbox_number">
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="group_center">
        <property name="title">
         <string>Center of rotation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_c_Z">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_c_X">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_c_Y">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="Gui::QuantitySpinBox" name="spinbox_c_X">
             <property name="unit" stdset="0">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="Gui::QuantitySpinBox" name="spinbox_c_Y">
             <property name="unit" stdset="0">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="Gui::QuantitySpinBox" name="spinbox_c_Z">
             <property name="unit" stdset="0">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="hbox">
        <item>
         <widget class="QCheckBox" name="checkbox_fuse">
          <property name="toolTip">
           <string>If checked, the resulting objects in the array will be fused if they touch each other</string>
          </property>
          <property name="text">
           <string>Fuse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
